.github/labels.yml

```
@@ -47,6 +44,9 @@
 - name: release-notes
   description: Issue/PR needs to be evaluated for inclusion in release notes highlights or upgrade instructions
   color: fb3c99
+- name: dependencies
```

I suggest using singular, like in the other items.
```suggestion
- name: dependency
```